packages/core/src/__tests__/feature-app-manager.test.ts

```diff
@@ -160,7 +160,12 @@ describe('FeatureAppManager', () => {
       );
 
       expect(mockFeatureAppCreate.mock.calls).toEqual([
-        [{config: mockConfig, featureServices: {}}]
+        [
+          {
+            config: mockConfig,
+            requiredFeatureServices: mockFeatureServicesBinding.featureServices
```

Comment:
Sorry to be pedantic, but this would still lead to false positive tests, if fore some reason we would always set an empty object as `requiredFeatureServices` in the implementation. Additionally to this change, you also need to add a property to `mockFeatureServicesBinding.featureServices` in https://github.com/sinnerschrader/feature-hub/pull/165/commits/127c0b1c5045b9a23cea5d0d64ec20240d2e1f5e#diff-2aa666ad2c64b8a6beaf6877bd973747R32 (because of `toEqual`).